flybywiresim/a32nx
# fix(mcdu): Import cost index into MCDU when importing from SimBrief

Fixes #3644

## Summary of Changes

Made sure the cost index set variable is set when importing from SimBrief.

## Screenshots (if necessary)



## References





## Additional context



Discord username (if different from GitHub): Avian#5239

## Testing instructions

- Import a flight plan from SimBrief and make sure the cost index is correct
- Create a normal flight plan from inside the simulator and make sure you can still enter cost index manually


## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created and uploaded.
The build script will have already been run with the latest changes, so no need to rerun it once you download the zip.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the right side, click on the **Artifacts** drop down and click the **A32NX** link
